corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -669,6 +669,9 @@ def _get_asynchronous_payload(self):
             # recorded since it is async (see self.get_response).
             with self.timing_context("wait_for_task_to_start"):
                 task = get_async_restore_payload.delay(self, self.domain, self.restore_user.username)
+                logging.info('RestoreConfig after get_async_restore_payload task is created: {msg}'.format(
```

Comment:
```suggestion
                logger.info('RestoreConfig after get_async_restore_payload task is created: {msg}'.format(
```

Reply:
nit: the logging package is pre-wired to do %-style string formatting:
```py
logger.info("message: %s", msg)
```
The cool thing about this is that string formatting is deferred until it is known that the log message will actually be written somewhere. So if the logging level is set to ERROR or WARNING (meaning INFO messages are ignored), then the string formatting would not be done for this example, which saves some time and processing resources. Unfortunately this does not work for `string.format()`-style formatting.

So you could make this a bit more efficient by changing it to
```py
logging.info('RestoreConfig after get_async_restore_payload task is created: %r', self)
```

Reply:
Nice. Thank you!